integration_tests/suite/base/test_incalls.py

```diff
@@ -165,6 +165,35 @@ def test_get(incall):
     )
 
 
+@fixtures.incall(id=0)
+@fixtures.user(id=1)
+@fixtures.incall(id=1)
+@fixtures.user(id=2)
+def test_get_userid(incall, user1, *_):
+    with a.incall_user(incall, user1):
+        response = confd.incalls().get()
+        assert_that(response.total, 2)
+
+        response = confd.incalls().get(user_id=user1['id'])
+        assert_that(response.total, 1)
+        assert_that(
+            response.items[0],
+            has_entry(
+                'destination',
+                has_entries(
+                    {
+                        'moh_uuid': None,
+                        'user_id': user1['id'],
+                        'ring_time': None,
+                        'type': 'user',
+                        'user_firstname': user1['firstname'],
+                        'user_lastname': user1['lastname'],
+                    }
+                ),
```

Comment:
```suggestion
                has_entries(user_id=user1['id']),
```
This test should focus to only one thing, i.e. that result return the matches user_id

If you want to test if all arguments are there, use `test_get_user_destination_relation` (we can see there is some arguments missing in this test)